### PR TITLE
Update runtime and modules

### PR DESCRIPTION
**Pull request by flyingcakes85**

### CODE REVIEW

File: org.kde.akregator.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "id": "org.kde.akregator",
     "runtime": "org.kde.Platform",
-    "runtime-version": "6.7",
+    "runtime-version": "6.9",
```
Comment (travier) [inline org.kde.akregator.json:4]: This one does not exists yet.

Comment (flyingcakes85) [reply org.kde.akregator.json:4]: Will rebuild when its available :+1:

**Comment by travier** [comment]
We should probably do https://github.com/flathub/org.kde.akregator/pull/182 first (6.8 runtime update)

File: org.kde.akregator.json (hunk 1 above)
Comment (tsdgeos) [reply org.kde.akregator.json:4]: I don't see the point in having this 3 months open in advance

**Pull request closed without merging**